content/_partners/faq._en.md

```diff
@@ -73,59 +69,33 @@ logistics_accordion:
     -
         id: b-3
-        title: What is the timeline for a IAA modifications?
+        title: What is identity verification?
         content: >-
-            Six to eight weeks. 
+            “Identity proofing is the process by which a CSP (credentialing service provider) collects, validates, and verifies information about a person.” NIST SP 800-63-3, Digital Identity Guidelines This is the process Login.gov uses to verify that a user is who they say they are. While many agencies can validate an individual’s identity through an in-person proofing experience, we developed an online application that allows individuals to have their identities verified from their smartphone or computer.
     -
         id: b-4
-        title: Does Login.gov provide authorization?
-        content: >-
-            Login.gov does not provide authorization. At this time, Login.gov supports authentication and identity proofing capabilities. We encourage agencies to take the lead on determining the best strategy for their role management and authorization. Our industry partners can help develop or provide existing solutions that can address your authorization needs.
-    -
-        id: b-5
-        title: Does Login.gov meet the NIST 800-63 standards for Identity Assurance Levels (IAL) and Authenticator Assurance Levels (AAL)?
+        title: Can Login.gov authenticate or verify the identity of non-U.S. citizens or non-U.S. immigrants (not a U.S. citizen, U.S. national, lawful permanent resident, or traveling to the United States on an immigrant visa)?
         content: >-
-            For our Login.gov basic authentication accounts (IAL1), we rely on the user having access to an email address, password, and a secure multi-factor authentication method (AAL2 or higher) such as a phone, authentication app or PIV/CAC where they can receive a secure code to use to sign in to their account.
-            <br/>
-            <br/>
-            For identity proofing, in addition to meeting the above requirements for IAL1/AAL2, we ask users to upload a photograph of their state-issued ID and share their address, phone number and other personal information which is then verified against authoritative sources. Login.gov’s proofing process is based on the IAL2 specifications but does not fully conform to the requirements—specifically biometrics and liveness check.
+            <p>Non-U.S. citizens and non-U.S. immigrants can authenticate with Login.gov, though select features (e.g., SMS / voice OTP for MFA) may be restricted in certain countries. Check our [International phone number support](/help/manage-your-account/international-phone-support/){:class="external-link"} for a complete list that Login.gov supports for authenticating end-users.</p>
```

Comment:
This link isn't parsing into a clickable link on the build page. See screenshot below
![Uploading Screen Shot 2022-08-04 at 2.35.05 PM.png…]()

Reply:
Oh and here's that rogue "Contact an account manager to get started" link again, I saw that in Nadya's PR as well. Seems like it needs to be taken out of the related html file.

Reply:
Ah! thanks for catching that. It's because it was in <p> tags and that was causing it to render from html.